src/components/AddPlaidBankAccount.js

```diff
@@ -16,15 +15,15 @@ import themeColors from '../styles/themes/default';
 import compose from '../libs/compose';
 import withLocalize, {withLocalizePropTypes} from './withLocalize';
 import Picker from './Picker';
-import plaidDataPropTypes from '../pages/ReimbursementAccount/plaidDataPropTypes';
+import {plaidDataPropTypes} from '../pages/ReimbursementAccount/plaidDataPropTypes';
 import Text from './Text';
 import getBankIcon from './Icon/BankIcons';
 import Icon from './Icon';
 import FullPageOfflineBlockingView from './BlockingViews/FullPageOfflineBlockingView';
 
 const propTypes = {
     /** Contains plaid data */
-    plaidData: plaidDataPropTypes,
+    plaidData: plaidDataPropTypes.isRequired,
```

Comment:
Getting a strange error related to the plaidDataPropTypes when on desktop:

![Screenshot 2023-01-11 at 2 47 27 PM](https://user-images.githubusercontent.com/12268372/211788507-e1dd35bd-0ca6-482b-9abc-c94e3c568a72.png)

But it doesn't seem to happen on web, strangely

Reply:
Can you `console.log('this.props.plaidData', JSON.stringify(this.props.plaidData));` in the render of BankAccountPlaidStep?
When does this error occur? When clicking on the Plaid option?

Reply:
Ah, can't seem to reproduce this error anymore but it was when clicking on `Connect back account`

Reply:
During my multiple testings, I think I also got it once, but then it disappeared.
So, I can't reproduce, but I feel like it might be due to stale data or Onyx update sync issue. So [that](https://github.com/Expensify/App/pull/13236/commits/87b322d49d5181ac6917bc8d14c185eeddb67b8b) might fix it (making sure PlaidData is reset to the right format before trying to display the view)?